src/EnergyPlus/SolarShading.cc

```diff
@@ -5970,7 +5970,24 @@ namespace SolarShading {
 					// The inside reveals receive solar (reflected part + absorbed part) from the window, this amount should be
 					// deducted from the BTOTZone, then adds the InsRevealDiffIntoZone
 					if ( SurfaceWindow( SurfNum ).WindowModelType == WindowBSDFModel ) { //Complex Fenestration
-						SurfSolIncPtr = SurfaceScheduledSolarInc( BackSurfaceNumber, ConstrNumBack );
+						// Jason Glazer added the following code and note on March 16, 2015 to address Github #4777
+						//
+						// Originally this just had the line:
+						//
+						//   SurfSolIncPtr = SurfaceScheduledSolarInc( BackSurfaceNumber, ConstrNumBack );
+						//
+						// prior to the line: if ( SurfSolIncPtr == 0 ) {
+						//
+						// But since BackSurfaceNumber, ConstrNumBack have not been set, added the code to scan through the
+						// back surfaces to look for the value. Since there can be more than one back surface for each, just
+						// loop through it and use the final value.
+						SurfSolIncPtr = 0;
+						int NBkSurf = ShadowComb( SurfNum ).NumBackSurf;
+						for ( int IBack = 1; IBack <= NBkSurf; ++IBack ) {
+							BackSurfaceNumber = ShadowComb( SurfNum ).BackSurf( IBack );
+							ConstrNumBack = Surface( BackSurfaceNumber ).Construction;
+							SurfSolIncPtr = SurfaceScheduledSolarInc( BackSurfaceNumber, ConstrNumBack );
```

Comment:
No. It is not solar absorptance values. It is incident solar radiation on interior side of the surface.
EnergyPlus calculates what is incident on each surface during each timestep and scheduled surface gains (which are external files) should overwrite what is calculated by EnergyPlus. This is where SurfSolIncPtr == 0 comes into play. If scheduled surface gain for some surface exists then EnergyPlus calculations should not be taken, but only what is provided in external schedule file. So SurfSolInc == 0 allows those calculations to be taken into account since schedule file does not exist.
Diffuse solar radiation into zone is calculated by multiplying incoming solar radiation with surface reflectance which happens later in the code.